### DISCUSSION TITLE
Add CreatedAt property

### DISCUSSION THREAD

**Discussion by romainvandooren** [bwmarrin/discordgo#1405] [Q&A]
Currently there is no way to get the user account creation date.

**Comment by AdamBrianBright** [comment]
If I'm not mistaken, Discord uses Snowflake ID for (almost?) everything, including user ID.
Take mine for example: `241129119688032257`. Try to parse it and convert to a timestamp and you'll get `2016-10-27T09:21:02.096Z` and if I open my discord it'll say "Member since Oct 27, 2016". 

So to get CreatedAt of a user, guild, channel, message, all you need is to parse it's ID:

```go
package main

import (
	"fmt"
	"github.com/bwmarrin/discordgo"
)

func main() {
	createdAt, err := discordgo.SnowflakeTimestamp("241129119688032257")
	if err != nil {
		panic(err)
	}
	fmt.Println(createdAt)
}
```

And as long as you are SURE that id is correct and not empty, you can skip error checking.

**Reply by AdamBrianBright** [reply]
But one can argue that it could be more obvious, better documented, or there could be an adapter method, that will parse ID for you...
